src/core/variables/ColorVariable.ts

```diff
@@ -71,6 +71,16 @@ export class ColorVariable extends Variable implements IColorVariableParams {
   }
 
   /**
+   * Updates the selected value.
+   * @override
+   * @param {any} value The selected value.
+   */
+  updateValue(value: any): void {
```

Comment:
why not just override the setter for selectedValue? having both a setter for selectedValue and an updateValue is a little confusing imo